src/CONST.ts

```diff
@@ -1264,7 +1264,6 @@ const CONST = {
     PUSHER: {
         PRIVATE_USER_CHANNEL_PREFIX: 'private-encrypted-user-accountID-',
         PRIVATE_REPORT_CHANNEL_PREFIX: 'private-report-reportID-',
-        PRESENCE_ACTIVE_GUIDES: 'presence-activeGuides',
```

Comment:
Same, are you sure we need to remove this value? It was added [not so long ago](https://github.com/Expensify/App/pull/48696/files), maybe it's a part of the feature under development